frontend/src/reducers/plannerReducer.js

```diff
@@ -38,6 +38,35 @@ if (planner) initialState = extractFromLocalStorage(planner);
 
 const plannerReducer = (state = initialState, action) => {
   let hidden, areYearsHidden;
+
+  const unscheduleCourse = (code) => {
+    let updatedUnplanned = state.unplanned;
+    updatedUnplanned.push(code);
+    const termTag = state.courses.get(code).plannedFor;
+
+    const yearI = parseInt(termTag.slice(0, 4)) - state.startYear;
+    const termI = termTag.slice(4);
+
+    const nTerm = new Object(
+      state.years[yearI][termI].filter((course) => course !== code)
```

Comment:
I'm not sure what's with the use of the `new` keyword so often. I see stuff like `new Map` too. I know you didn't write this code, but couldn't this be written more expressively with using {} notation? I know this is a nitpick, so ill approve you anyways if you find this easier to read.

Reply:
ahh righto I see, will have a look